subsys/bluetooth/controller/ll_sw/lll_sync.h

```diff
@@ -34,6 +34,7 @@ struct lll_sync {
 
 	uint8_t phy:3;
 	uint8_t is_rx_enabled:1;
+	uint8_t cte_type:4;
```

Comment:
Conditional compile?

Reply:
Why conditionally compile? It is a feature of per. adv. sync not a feature of direction finding.

Reply:
Then:
1. Whole feature should configurable to enable/disable.
2. The code may be left as is, because the BT Core Spec 5.0 reserves the Sync_CTE_Type (as `Unused`) parameter of HCI_LE_Periodic_Advertising_Create_Sync command. It is imposed on host to always send value zero in this parameter  to comply with the 5.0 Core spec. So it would be handled as no filtering by this implementation.
Besides bigger code and more instructions executed behavior should be the same.